playwright/_impl/_element_handle.py

```diff
@@ -56,57 +56,68 @@ def __init__(
     ) -> None:
         super().__init__(parent, type, guid, initializer)
         self._frame = cast("Frame", parent)
-        self._channel._set_timeout_calculator(self._frame._timeout)
 
     async def _createSelectorForTest(self, name: str) -> Optional[str]:
-        return await self._channel.send("createSelectorForTest", dict(name=name))
+        return await self._channel.send(
+            "createSelectorForTest", self._frame._timeout, dict(name=name)
+        )
 
     def as_element(self) -> Optional["ElementHandle"]:
         return self
 
     async def owner_frame(self) -> Optional["Frame"]:
-        return from_nullable_channel(await self._channel.send("ownerFrame"))
+        return from_nullable_channel(
```

Comment:
Most methods in this class do not have a timeout, but some do. Please align with Node.js code to avoid any changes in behavior.